index.html

```diff
@@ -5,14 +5,69 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Spotify Clone</title>
-    <!-- don't forget to link your styles -->
+    <link rel="stylesheet" href="./styles/style.css">
   </head>
+
   <body>
-    Premium Discover Help Download Music for everyone. Spotify is now free on mobile, tablet and computer. Listen to the
-    right music, wherever you are. What’s on Spotify? Millions of Songs There are millions of songs on Spotify HD Music
-    Listen to music as if you were listening live Stream Everywhere Stream music on your smartphone, tablet or computer
-    It’s as yeezy as Kanye West. Search Know what you want to listen to? Just search and hit play. Browse Check out the
-    latest charts, brand new releases and great playlists for right now. Discover Enjoy new music every Monday with your
-    own personal playlist. Or sit back and enjoy Radio.
+    <header class="header">
+      <div class="nav">
+        <img class="header-icon" src="./images/spotify-logo.png" alt="logo spotify">
+        <div class="header-list">
+          <ul>
+            <li>Premium</li>
+            <li>Discover</li>
+            <li>Help</li>
+            <li>Download</li>
+          </ul>
+        </div>
+      </div>
+      <div class="header-text">
+        <h1>Music for everyone.</h1>
+        <p>Spotify is now free on mobile, tablet and computer. <br> Listen to the
+          right music, wherever you are.</p>
+      </div>
+    </header>
+
+    <section class="spotify-offers">
+      <h2>What’s on Spotify?</h2>
+      <div class="container">
+        <div class="container-box">
+          <img src="./images/music-icon.png" alt="music-icon">
+          <h3>Millions of Songs</h3>
+          <p>There are millions of songs on Spotify</p>
+        </div>
+        <div class="container-box">
+          <img src="./images/high-quality-icon.png" alt="quality">
+          <h3>HD Music</h3>
+          <p>Listen to music as if you were listening live </p>
+        </div>
+        <div  class="container-box">
+          <img src="./images/devices-icon.png" alt="Stream">
+          <h3>Stream Everywhere</h3>
+          <p>Stream music on your smartphone, tablet or computer</p>
+        </div>
+      </div>
+    </section>
+
+    <footer>
+      <img class="footer-logo" src="./images/spotify-icon-white.png" alt="spotify-logo white">
+      <div class="footer-last">
+        <main class="text">
+          <h2>It’s as yeezy as Kanye West.</h2>
+          <h3>Search</h3>
+          <p>Know what you want to listen to? <br>Just search and hit play.</p>
+          <h3>Browse</h3>
+          <p>Check out the latest charts, <br> brand new releases and great <br> playlists for right now.</p>
+          <h3>Discover</h3>
+          <p>Enjoy new music every Monday <br> with your own personal playlist. <br>Or sit back and enjoy Radio.</p>
+        </main>
+        <main class="image">
```

Comment:
Maybe  _image_ here as class name can be a little bit confusing. I would use _image-container_